src/AWS.Distro.OpenTelemetry.AutoInstrumentation/Plugin.cs

```diff
@@ -401,9 +420,22 @@ private bool IsApplicationSignalsEnabled()
         return System.Environment.GetEnvironmentVariable(ApplicationSignalsEnabledConfig) == "true";
     }
 
+    private bool IsApplicationSignalsRuntimeEnabled()
+    {
+        return false;
```

Comment:
The next PR will enable the feature by default: https://github.com/aws-observability/aws-otel-dotnet-instrumentation/pull/140

Right now, we don't turn the runtime metrics on because it depends on new changes in CWA.